### ISSUE TITLE
bookmark sidebar links disabled under certain circumstances

### ISSUE THREAD

**Issue by jupester** [sumatrapdfreader/sumatrapdf#2465]
Hi, this is something I have hoped would get fixed for a while but have decided to report it since it might have been going unnoticed.  What the issue I am experiencing is that a certain pdf I use frequently, when I click on the bookmark link in the left hand panel, once I have clicked on it once, if I subsequently then click on a link in the right hand panel that is within the range of the bookmark I can then no longer click on the bookmark to return back to the first bookmarked page.
To explain it clearer see the below image, if I click on the "Scripting Commands Reference" it will take me to that bookmark ok, then if I click on one of the links that the red arrow is pointing to on the right pane they will take me to another part of the pdf, but now I can no longer click on the "Scripting Commands Reference" bookmark again, which I need to do to get back to the index.  In older versions of Sumatra this was possible, it then stopped working in I think from memory one of the pre v3.3.3 betas then it started working again for a few beta releases, and now it no longer works and hasn't for probably over a year now.

Hopefully this something you can make work again, and it wasn't an intentional change.

![Sumatra Bug 2022-04-04_112513](https://user-images.githubusercontent.com/50538846/162335045-a5e9b077-2117-4989-9eba-29869783f74a.png)

Here is a link to an example pdf which presents the issue: https://www.xyplorer.com/download/XYplorerHelp.pdf

I am using the latest 3.4.14330 pre-release.

**Comment by GitHubRulesOK** [comment]
Hmmm
 I seem to remember there was some dead cat bouncing in that area
so ALT LEFT ARROW is the way back however you are right you need
to click on any other one then go back to click on the desired so it works

**Comment by jupester** [comment]
Yeah I am aware of the workarounds since I have been enduring this issue for a couple years, it would be great if they weren't required and it could work like it used to though, with the bookmark activated even if you are already on it.

**Comment by kjk** [comment]
Yes, it's a subtlety in `TreeView` handling.

Technically `Scripting Commands Reference` items is already selected so when user clicks on it, we don't get "selection changed" event.

We also need to add handling for "mouse click on item" event to act even if selection technically didn't change.
